### PR TITLE
feat(avo-2648): add kliks tab and page, replace links

### PR DESCRIPTION
**Pull request by SHDBramVandenbussche**
[AVO-2648](https://meemoo.atlassian.net/browse/AVO-2648)
[AVO-2649](https://meemoo.atlassian.net/browse/AVO-2649)

Before:
image

After: 
image

[AVO-2648]: https://meemoo.atlassian.net/browse/AVO-2648?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AVO-2649]: https://meemoo.atlassian.net/browse/AVO-2649?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by WardVerc1 — APPROVED** [review]

File: src/assignment/hooks/assignment-teacher-tabs.tsx [hunk 1]
```diff
@@ -38,6 +53,16 @@ export function useAssignmentTeacherTabs(): [
 	const onTabClick = useCallback(
 		(id: string | number) => {
```
Comment (bertyhell) [inline src/assignment/hooks/assignment-teacher-tabs.tsx:54]:
```suggestion
		(tabId: string | number) => {
```

File: src/assignment/views/AssignmentEdit.tsx [hunk 2]
```diff
@@ -90,6 +97,7 @@ const AssignmentEdit: FunctionComponent<DefaultSecureRouteProps<{ id: string }>>
 	const [assignment, setAssignment] = useAssignmentForm(undefined);
 	const [assignmentHasPupilBlocks, setAssignmentHasPupilBlocks] = useState<boolean>();
 	const [assignmentHasResponses, setAssignmentHasResponses] = useState<boolean>();
+	// const [tabId, setTabId] = useState<string | null>(null);
```
Comment (bertyhell) [inline src/assignment/views/AssignmentEdit.tsx:100]: remove comment?

File: src/assignment/views/AssignmentResponses.tsx [hunk 3]
```diff
@@ -590,33 +589,11 @@ const AssignmentResponses: FunctionComponent<AssignmentResponsesProps> = ({
...
-							</BlockHeading>
-						</Flex>
-					</Container>
-				</Container>
-
-				<Container mode="vertical" size="small">
-					<Container mode="horizontal">{renderAssignmentResponsesView()}</Container>
-				</Container>
-			</div>
+			// <div className="c-assignment-responses">
```
Comment (bertyhell) [inline src/assignment/views/AssignmentResponses.tsx:592]: remove comments?

File: src/constants.ts [hunk 4]
```diff
@@ -206,12 +206,12 @@ export const APP_PATH: { [routeId in RouteId]: RouteInfo } = {
...
 		showInContentPicker: false,
 		showForInteractiveTour: true,
 	},
-	ASSIGNMENT_RESPONSES: {
-		route: `/${ROUTE_PARTS.workspace}/${ROUTE_PARTS.assignments}/:id/${ROUTE_PARTS.responses}`,
+	ASSIGNMENT_EDIT_TAB: {
+		route: `/${ROUTE_PARTS.workspace}/${ROUTE_PARTS.assignments}/:id/${ROUTE_PARTS.edit}/:tabId`,
 		showInContentPicker: false,
 		showForInteractiveTour: false,
 	},
```
Comment (bertyhell) [inline src/constants.ts:217]: do we still need both or would one work as well?
```
	ASSIGNMENT_EDIT: {
		route: `/${ROUTE_PARTS.workspace}/${ROUTE_PARTS.assignments}/:id/${ROUTE_PARTS.edit}/:tabId`,
		showInContentPicker: false,
		showForInteractiveTour: true,
	},
```

**Review by bertyhell — APPROVED** [review]

**Pull request merged**